<launch>    
    <node name="rectbot_object_detection_node" pkg="rectbot_cv" type="rectbot_object_detection_node" output="screen">
            <!-- Load model parameters from config file -->
            <rosparam command="load" file="$(find rectbot_cv)/config/model_params.yaml" />
            <remap from="image" to="/camera/color/image_raw" />
            <param name="debug_view" value="true" />
            <!-- <param name="model/weights" value="$(find rectbot_cv)/models/yolov8n.engine" /> -->
        </node>

</launch>